### PR TITLE
Text Response submission view: answer input for students, solutions for staff in student view

### PR DESCRIPTION
**Pull request by makqien**
Reference #2630 (specifically f4a34483700b0d55638898114d83f511e72621a7).

Existing React view for students shows summernote input (should be plaintext input) for Text Response question despite having solution(s) being configured.

Existing React view for staff shows the solutions table (should be hidden) when Student View is enabled.

### CODE REVIEW

**Comment by cysjonathan** [comment]
image
Also, "student view" as instructor still displays the "solutions", not sure if intended behavior.

**Comment by benleong** [comment]
Thanks. Qi En will look into this.

File: client/app/bundles/course/assessment/submission/components/Answers.jsx [hunk 1]
```diff
@@ -144,14 +144,14 @@ export default class Answers extends Component {
...
+    const editableAnswer = question.autogradable ? plaintextAnswer : richtextAnswer;
 
     return (
       <div>
         { readOnly ? readOnlyAnswer : editableAnswer }
-        {question.solutions ? Answers.renderTextResponseSolutions(question) : null}
+        {question.solutions && graderView ? Answers.renderTextResponseSolutions(question) : null}
         {allowUpload ? <UploadedFileView questionId={question.id} /> : null}
-        {allowUpload && !readOnly ? Answers.renderFileUploader(question, readOnly, answerId) : null}
+        {allowUpload && !readOnly ? Answers.renderFileUploader(question, readOnly, answerId, graderView) : null}
```
Comment (jeremyyap) [inline client/app/bundles/course/assessment/submission/components/Answers.jsx]: `graderView` parameter is not needed

Comment (makqien) [reply client/app/bundles/course/assessment/submission/components/Answers.jsx]: Removed. Thank you!

**Pull request merged**